### PR TITLE
Feat/collection attributes

### PR DESCRIPTION
**Pull request by EdoWahdana**
## Changes
[x] Add Attributes button
[x] Show active filter attributes

## Concern
[x] Slow load on collection with abundant attributes
[x] Scroll Y snap to top when apply

### CODE REVIEW

File: pages/collection/[collection_id].js [hunk 1]
```diff
@@ -50,9 +52,17 @@ const CollectionPage = ({ collectionId, collection, serverQuery }) => {
...
+			params: {
+				collection_id: collectionId,
+			},
+		})
+
+		const newAttributes = await attributes.data.data.results
 		const newStat = await stat.data.data.results
 		const newData = await res.data.data
 		const newTokens = [...tokens, ...newData.results]
+		setAttributes(Array(newAttributes)[0])
```
Comment (ahnafalfariza) [inline pages/collection/[collection_id].js:65]: ini waktu itu kenapa mesti ditambah `Array()` ya do? kalau langsung `setAttributes(newAttributes)` bisa ga ya?

File: pages/collection/[collection_id].js [hunk 2]
```diff
@@ -280,20 +318,41 @@ const CollectionPage = ({ collectionId, collection, serverQuery }) => {
...
 									<FilterMarket isShowVerified={false} />
 								</div>
 							</div>
 						</div>
 					)}
 				</div>
-				<div className="mt-12 px-4">
+				<div className="relative flex flex-row flex-wrap left-0 ml-5 mt-5 ">
+					{router.query.attributes &&
+						router.query.tab != 'activity' &&
```
Comment (ahnafalfariza) [inline pages/collection/[collection_id].js:338]: `!==` ya

Comment (ahnafalfariza) [reply pages/collection/[collection_id].js:338]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Inequality
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_inequality

File: pages/collection/[collection_id].js (hunk 1 above)
Comment (EdoWahdana) [reply pages/collection/[collection_id].js]: Wah iya ini lupa difix waktu itu

**Pull request merged**